docs/ansible.netcommon.libssh_connection.rst

```diff
@@ -8,7 +8,7 @@ ansible.netcommon.libssh
 **(Tech preview) Run tasks using libssh for ssh connection**
 
 
-Version added: 2.10
+Version added: 2.6.0
```

Comment:
Shouldn't this be `1.0.0`?

Reply:
https://github.com/ansible-collections/ansible.netcommon/blob/main/CHANGELOG.rst#v110